README.md

```diff
@@ -63,3 +63,34 @@ resolve after running `mvn compile` at least once, then open up the javabot.iml
 
 To test the web application aspects, you need to copy the `javabot-sample.yml` to `javabot.yml`; this will put the
 web container on port 8081 by default.
+
+Testing on a Live Server
+------
+Freenode maintains a [test instance of their ircd](https://freenode.net/news/testing-the-nets).  This is ideal for quickly running javabot and testing out commands or verifying your configuration.
+
+To quickly get javabot running on a live server, ensure you've followed the steps in the Building section and change javabot.properties as follows:
+
+```
+# Connect to freenode test
+javabot.server=testnet.freenode.net 
+javabot.port=9002
+
+# Give your javabot a unique nick
+javabot.nick=tk-421
```

Comment:
Inspired by @tgolen

Reply:
Hahaha, I was quite surprised to see this pop-up in my inbox this morning. Well played sir o/